package.json

```diff
@@ -1,5 +1,4 @@
 {
-  "version": "3.47.3",
   "name": "store-components",
```

Comment:
Why was the version moved?

Reply:
I think you can just revert all changes to this file, `.eslintrc` and `prettierrc`